### PR TITLE
🌐 Add Chinese translation for Benchmarks

### PR DESCRIPTION
**Pull request by spaceack**
This PR translates the benchmarks.md to Chinese.
Please help review. @waynerv @Dustyposa @RunningIkkyu @Choyeon

### CODE REVIEW

File: docs/zh/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# 基准测试
+
+独立公司 TechEmpower 的基准测试表明在 Uvicorn 下运行的 **FastAPI** 应用程序是 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">可用的最快的 Python 框架之一</a>，仅次与 Starlette 和 Uvicorn 本身 (由 FastAPI 内部使用）。(*)
+
+但是在检查基准测试或与其它同类比较时，请注意以下几点。
+
+## 基准测试和速度
+
+当您做基准测试检查时，通常会看到被视为等效的几种不同类型的工具。
```
Comment (waynerv) [inline docs/zh/docs/benchmarks.md:9]: 当你查看基准测试时，几个不同类型的工具被等效地做比较是很常见的情况。

File: docs/zh/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+层次结构如下：
+
+* **Uvicorn**：ASGI服务器
+    * **Starlette**：（使用 Uvicorn）网络微框架
+        * **FastAPI**：（使用 Starlette） 具有多个附加功能的API微框架，用于构建API，进行数据验证等。
+
+* **Uvicorn**:
+    * 具有最佳性能，因为除了服务器本身外，它没有太多额外的代码。
+    * 您不会直接在 Uvicorn 中编写应用程序。这意味着您的代码至少必须包含 Starlette（或 **FastAPI**）提供的代码。如果您这样做了，最终应用程序和框架会最大程度地减少应用程序代码的编码量和错误。
```
Comment (waynerv) [inline docs/zh/docs/benchmarks.md:23]: ”And if you did that, your final application would have the same overhead as having used a framework and minimizing your app code and bugs.“ means "如果你这样做了（即直接在 Uvicorn 中编写应用程序），你最终的应用程序会和使用了框架并且最小化了应用代码和 bug 的情况具有相同的性能损耗。

File: docs/zh/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    * 如果要对比与 Uvicorn 对标的服务器，请将其与 Daphne，Hypercorn，uWSGI等应用服务器进行比较。
+* **Starlette**: 
+    * 在 Uvicorn 后使用 Starlette，性能会略有下降。实际上，Starlette 使用 Uvicorn运行。因此，由于必须执行更多的代码，它只会比 Uvicorn 更慢。
+    * 但它为您提供了构建简单的网络程序的工具，并具有基于路径的路由等功能。
+    * 如果想对比与 Starlette 对标的开发框架，请将其与 Sanic，Flask，Django 等网络框架（或微框架）进行比较。
+* **FastAPI**:
+    * 与 Starlette 使用 Uvicorn 一样，由于 **FastAPI** 使用 Starlette，因此 FastAPI 不能比 Starlette 更快。
+    * FastAPI 在 Starlette 基础上提供了更多功能。例如在开发 API 时，所需的数据验证和序列化功能。FastAPI 可以帮助您自动生成 API文档，（文档在应用程序启动时自动生成，所以不会增加应用程序运行时的开销）。
+    * 如果您不使用 FastAPI 而直接使用 Starlette（或诸如 Sanic，Flask，Responder 等其它工具），你则要自己实现所有的数据验证和序列化。那么最终您的应用程序会和使用 FastAPI 构建的程序有相同的开销。一般这种数据验证和序列化的操作在您应用程序的代码中会占很大比重。
+    * 因此，通过使用 FastAPI 意味着你可以节省开发时间，减少编码错误，用更少的编码实现其功能，如果您不使用 FastAPI，可能会获得同以前相同（或更好）的性能（因为您必须在代码中实现所有功能）。
```
Comment (waynerv) [inline docs/zh/docs/benchmarks.md:33]: "and you would probably get the same performance (or better) you would if you didn't use it (as you would have to implement it all in your code)." should be "并且相比不使用 FastAPI 你很大可能会获得相同或更好的性能（因为那样你必须在你的代码中实现所有相同的功能）

**Review by waynerv — CHANGES_REQUESTED** [review]

File: docs/zh/docs/benchmarks.md (hunk 2 above)
Comment (spaceack) [reply docs/zh/docs/benchmarks.md:23]: Thank you for your review

**Review by waynerv — APPROVED** [review]
LGTM.

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by tiangolo** [comment]
Great! Thank you @spaceack :bow: :rocket: 

And thanks @waynerv and @Laineyzhang55 for the reviews :clap: :clap: :coffee:

**Pull request merged**